src/webgpu/shader/execution/expression/call/builtin/textureSampleLevel.spec.ts

```diff
@@ -33,12 +34,100 @@ import {
   SamplePointMethods,
   skipIfTextureFormatNotSupportedOrNeedsFilteringAndIsUnfilterable,
   TextureCall,
+  vec1,
   vec2,
   vec3,
 } from './texture_utils.js';
 
 export const g = makeTestGroup(AllFeaturesMaxLimitsGPUTest);
 
+g.test('sampled_1d_coords')
+  .specURL('https://www.w3.org/TR/WGSL/#texturesamplelevel')
+  .desc(
+    `
+fn textureSampleLevel(t: texture_1d<f32>, s: sampler, coords: f32, level: f32) -> vec4<f32>
+
+Parameters:
+ * t  The sampled or depth texture to sample.
+ * s  The sampler type.
+ * coords The texture coordinates used for sampling.
+ * level The mip level, clamped to 0.
```

Comment:
Will `clamped to 0` be part of the spec? Do I need to do something about it in the CTS and Tint implementation?